app/(dashboard)/edit/[id]/page.tsx

```diff
@@ -13,6 +14,20 @@ const EditCvPage = async ({ params: { id } }: EditCvPageProps) => {
   const titles = (await fetchTitles()) || [];
   const cv = await fetchCv(id);
   const initialUserSkills = cv?.cv_skill || [];
+  const supabase = createServerComponentClient();
+
+  const {
+    data: { session },
+  } = await supabase.auth.getSession();
+
+  const userId = session?.user.id;
+
+  const { data } = await supabase
+    .from("admins")
+    .select("id")
+    .filter("user_id", "eq", userId);
+
+  const isAdmin = data?.length && data.length > 0 ? true : false;
```

Comment:
I guess this is the same as

```suggestion
  const isAdmin = data?.length > 0;
```

Reply:
not really cuz the suggested line can be undefined if the data is undefined resulting in a ts error

Reply:
what about const `isAdmin = !!data?.length;` ?

Reply:
That would work, but I've pushed already change, used ternary operator
```typescript
const isAdmin = data ? data.length > 0 : false;
```